### ISSUE TITLE
Combine function show up on 01-rstudio-intro

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/r-novice-gapminder#189]
On lesson 01-rstudio-intro, the answer to the last question is

> ```
> install.packages(c("ggplot2", "plyr", "gapminder"))
> ```

but we didn't introduce the combine or `c``function.

I suggest that we replace the answer with

> ```
> install.packages(c("ggplot2"))
> install.packages(c("plyr"))
> install.packages(c("gapminder"))
> ```

**Comment by naupaka** [comment]
Why not: 

```r
install.packages("ggplot2")
install.packages("plyr")
install.packages("gapminder")
```

If you include `c()` anyway it still a problem.

PR would be welcome to tweak this.

**Comment by naupaka** [comment]
Fixed by #229
